### PR TITLE
Expired trial apps

### PR DESCRIPTION
**Pull request by alexweininger**
When a Trial app is expired the only child is a "Transfer to Subscription..." node.

image

And the only actions in the right click context menu are:
image

### CODE REVIEW

File: src/explorer/trialApp/TrialAppTreeItem.ts [hunk 1]
```diff
@@ -96,6 +99,9 @@ export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean, _context: IActionContext): Promise<AzExtTreeItem[]> {
+        if (isNaN(this.minutesLeft)) {
```
Comment (nturinski) [inline src/explorer/trialApp/TrialAppTreeItem.ts:102]: I think we should just have a property on the trialAppTreeItem that is a boolean called `isExpired`.  You'll have to update it in refres, but then you don't have to have that isNaN check everywhere.

Comment (alexweininger) [reply src/explorer/trialApp/TrialAppTreeItem.ts:102]: I added it to `TrialAppClient`. I think that's the best place for it. I need to wait about 30 minutes to test it...

Comment (nturinski) [inline src/explorer/trialApp/TrialAppTreeItem.ts:102]: Missed this one

File: src/explorer/trialApp/TrialAppTreeItem.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { AppSettingsTreeItem, DeploymentsTreeItem, LogFilesTreeItem, SiteFilesTreeItem } from 'vscode-azureappservice';
+import { ext } from 'vscode-azureappservice/out/src/extensionVariables';
```
Comment (nturinski) [inline src/explorer/trialApp/TrialAppTreeItem.ts:7]: Wrong ext was auto imported here.

File: src/explorer/trialApp/TrialAppTreeItem.ts [hunk 3]
```diff
@@ -96,6 +99,9 @@ export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean, _context: IActionContext): Promise<AzExtTreeItem[]> {
+        if (this.client.isExpired) {
+            return [new GenericTreeItem(this, { label: 'Transfer to Subscription...', commandId: `${ext.prefix}.TransferToSubscription`, contextValue: 'transferToSubscription' })];
```
Comment (nturinski) [inline src/explorer/trialApp/TrialAppTreeItem.ts:103]: Could you localize the label string please? 🙏

**Pull request merged**